bot/seasons/evergreen/snakes/utils.py

```diff
@@ -623,11 +616,12 @@ def game_event_check(reaction_: Reaction, user_: Member):
                 if reaction.emoji == ROLL_EMOJI:
                     await self.player_roll(user)
                 elif reaction.emoji == CANCEL_EMOJI:
-                    if self.ctx.author == user:
-                        await self.cancel_game(user)
+                    if self._is_moderator(user) and not self._mod_in_game_check(user):
```

Comment:
Since the first condition checks that they are a moderator, using `_mod_in_game_check()` has some redundancy.
```suggestion
                    if self._is_moderator(user) and user not in self.players:
```